sass/directives/_headers.scss

```diff
@@ -44,30 +24,58 @@
 
     @media only screen and (max-width: $small-screen-max) {
       position: relative;
+      padding-top: $header-top-bottom-padding;
+      padding-bottom: $header-top-bottom-padding;
     }
 
     h1 {
       margin-top: 0;
+      margin-bottom: $header-headline-bottom-margin;
       line-height: normal;
-      color: $hero-text-color-dark;
+      color: $hero-text-color;
+
+      @media only screen and (max-width: $small-screen-max) {
+        margin-bottom: 0;
+      }
     }
 
     p {
+      max-width: 28em;
       margin-bottom: 0;
       font-size: $hero-text-size;
-      color: $hero-text-color-dark;
+      color: $hero-text-color;
     }
   }
 }
 
-@mixin header-hero {
+@mixin header-hero--home($background-image-large, $background-image-medium: nil) {
   position: relative;
+  min-height: $header-height-large;
   overflow: hidden;
   background-color: $base-background-color;
+  background-image: url(asset_path(#{$background-image-large}));
+  background-size: auto $header-height-large;
+  background-position: right bottom;
+  background-repeat: repeat no-repeat;
+
```

Comment:
Not too much of a biggy, since breaking out the background styles increases readability for this particular element, but for other applications, this is something that could also be condensed to a single line. I'll leave this up to you if you want to keep it as is or condense it.